testing/tests/DevExpress.ui.widgets/accordion.tests.js

```diff
@@ -233,6 +234,46 @@ QUnit.test("nested widget rendering", function(assert) {
     assert.equal(nested.isItemSelected(1), true, "item selected by click");
 });
 
+QUnit.test("nested widget with onItemTitleClick", function(assert) {
+    var that = this,
+        isFired = false,
+        nested;
+
+    var handleFire = function() {
```

Comment:
You can use [sinon.stub](https://sinonjs.org/releases/v7.0.0/stubs/) and check `callCount` or `notCalled` property

```
var titleClickHandler = sinon.stub();

$element.dxAccordion({
  //config
  onItemTitleClick: titleClickHandler
});

//actions

assert.ok(titleClickHandler.notCalled);